phase1/azure/Kconfig

```diff
@@ -8,17 +8,17 @@ config phase1.azure.subscription_id
 config phase1.azure.tenant_id
 	string "Azure TenantID"
 	help
-		The Azure Tenant ID
+		The Azure Tenant ID. (If left blank, will be auto-determined from the Subscription ID)
```

Comment:
It's just a bit awkward otherwise. If I remove it completely, then the json file will be in different shapes at different stages. After the Kconfig wizard, it will just be missing that field, but then one it hits `do`, it will get merged back in.

Maybe it's not so bad to have that be the case... I'll try it out and report back.